tests/test_tensor_interface.py

```diff
@@ -0,0 +1,159 @@
+import pytest
+
+import aesara
+import aesara.sparse as sparse
+import aesara.tensor as at
+
+
+@pytest.mark.parametrize("is_sparse", [False, True])
```

Comment:
Does the non-sparse case need to be tested explicitly?  It seems like we would necessarily have coverage for the non-sparse/dense cases via the existing `aesara.tensor` tests.